src/com/sun/ts/tests/websocket/ee/jakarta/websocket/websocketmessage/WSFullShortAndSessionServer.java

```diff
@@ -30,7 +30,7 @@
 public class WSFullShortAndSessionServer {
 
   @OnMessage
-  public String echo(Short s, Session session) {
```

Comment:
I know the session is not used. But the test is about the method with arguments Short and Session being used. See the class name: `WSFullShortAndSessionServer`. There is another endpoint `WSFullShortServer.java` that has ``echo(Short s)`. So now you have effectively duplicated a test and removed a test that verifies that a method with `Session` argument is invoked at all.

Reply:
Thanks. I thought I caught all of those but I must have missed that one. I'll update that part of the clean-up and re-check the rest..
Note: I'm still happy to move the clean-up to a separate PR. It is the fix for the slow/unstable tests that is important.

Reply:
It looks like this is removed by mistake, all other similar cases are unchanged

Reply:
Yep, good work on all the tests with CountDownLatch(too_many).